playbooks/common/openshift-cluster/redeploy-certificates/ca.yml

```diff
@@ -294,7 +294,7 @@
       client_path: "{{ openshift.common.config_base }}/master/admin.kubeconfig"
       ca_path: "{{ openshift.common.config_base }}/master/ca-bundle.crt"
   - name: Lookup default group for ansible_ssh_user
-    command: "/usr/bin/id -g {{ ansible_ssh_user }}"
+    command: "/usr/bin/id -g {{ ansible_ssh_user | quote }}"
```

Comment:
It looks like you can use the `user` module instead of command here:
```
- name Lookup default group for ansible_ssh_user
  user:
    name: ansible_ssh_user
  register:l _ansible_ssh_user
```
And then access it with: `{{ l_ansible_ssh_user.group }}`